mysql-test/suite/rocksdb/t/lock_wait_timeout_stats.test

```diff
@@ -0,0 +1,34 @@
+create table t (a int not null primary key) engine=rocksdb;
```

Comment:
nit: the not null is redundant here since it PRIMARY KEY always have the not null constraint.